mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Rights.test

```diff
@@ -38,3 +38,12 @@ S: a6 NO DELETEACL failed. Mailbox not found.
 
 C: a7 SETACL INBOX bob lr
 S: a7 OK SETACL completed.
+
+C: a8 SETACL INBOX anyone p
+S: a8 OK SETACL completed.
+
+C: a8 SETACL INBOX anyone -p
+S: a8 OK SETACL completed.
+
+C: a8 SETACL INBOX -bob p
+S: a8 OK SETACL completed.
```

Comment:
What if I `SETACL bob p` and then after `SETACL -bob p` what is the final state of the ACL? is bob allowed?

Idem with the reverse.